py-polars/polars/dataframe/frame.py

```diff
@@ -5764,7 +5766,7 @@ def clear(self, n: int = 0) -> Self:
 
         See Also
         --------
-        clone : Cheap deepcopy/clone.
+        clone : Cheap deepcopy/clone of schema (no rows copied).
```

Comment:
This isn't correct; `clone` is not a copy of the schema.

Reply:
Yes, I think we should leave this text short. As the explanation should be on the docstring of that function. (and rows are never copied as we don't have that memory construct).

Reply:
What do you mean "clone doesn't copy the scheme?" That's all it copies (i.e., it doesn't copy any rows). The docs should say that. 

Why do you want the doc string to be short. I want the doc string, which is compiled into the end-user docs, to fully explain what you need to know to use the function.

Reply:
> Why do you want the doc string to be short. I want the doc string, which is compiled into the end-user docs, to fully explain what you need to know to use the function.

Because this isn't the docstring of clone. This is the see also reference.

The explanation is and should be in the docstring of clone.

Reply:
On further review, the update should go into to docstring of `clone(...)`. It can then be reflected here, as matching the description line of the docstring.

Reply:
> What do you mean "clone doesn't copy the schema?"

The schema is a property of a DataFrame that is associated with it and serves to describe it - it is not, itself, a DataFrame. Clone operates on a DataFrame, not on the schema associated with a DataFrame ;)